package.json

```
@@ -14,25 +14,35 @@
     "ms": "^2.1.1"
   },
   "devDependencies": {
+    "@eslint/js": "^9.15.0",
     "@types/github-url-to-object": "^4.0.1",
     "@types/jest": "^29.5.14",
     "@types/ms": "^0.7.31",
-    "electron": "^22.0.0",
+    "@types/node": "^22.10.1",
+    "electron": "^33.2.1",
+    "eslint": "^9.15.0",
+    "eslint-config-prettier": "^9.1.0",
+    "husky": "^9.1.7",
     "jest": "^29.0.0",
+    "lint-staged": "^15.2.10",
     "prettier": "^3.0.3",
-    "standard": "^14.3.4",
-    "standard-markdown": "^6.0.0",
     "ts-jest": "^29.2.5",
-    "typescript": "^4.9.4"
+    "typescript": "^5.7.2",
+    "typescript-eslint": "^8.16.0"
   },
   "scripts": {
-    "prepare": "tsc",
-    "test": "jest && standard --fix && standard-markdown",
+    "lint": "npx eslint src test && npx prettier . --check",
+    "prepare": "tsc && husky install",
```

```suggestion
    "prepare": "tsc && husky",
```

Can drop the `install` on `husky install` with Husky v9.

---

[cf9a424](https://github.com/electron/update-electron-app/pull/172/commits/cf9a424ff4ef21721907db3381471449328110a0)